packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -122,20 +122,16 @@ export const TranslationSidebar = ({
     bdbBookRefNames[bookId - 1]
   } ${chapterNumber}:${verseNumber}`;
 
-  const lexiconEntryHTML = useMemo(() => {
+  const lexiconEntryRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
     if (isHebrew) {
```

Comment:
I think we can remove this condition. Yes, the LSJ doesn't have verses in its content, but the DOM query will come up empty every time and this will be less complicated.

Reply:
Done.